2023/did-wg.html

```diff
@@ -182,10 +179,7 @@ <h2>Scope</h2>
         </ul>
 
         <p>
-The Working Group may also publish new Working Group Notes, Editors Drafts, or
-Working Drafts for DID Resolution or DID URL Dereferencing, or of DID Method
-Specifications. Any new documents intended for Recommendation will be
-limited to the Working Draft maturity level.
+The Working Group may also publish new Working Group Notes.
```

Comment:
Isn't this always true?

Reply:
It is not 100% clear. The only thing the [process document](https://www.w3.org/2023/Process-20230612/#note-track) says is that (paraphrasing the reference above) WG Notes may be published by WGs only (as opposed to other types of groups), but that does not say that a specific group is indeed allowed to do so without further ado. To avoid further unnecessary discussions, I think it is wise to keep this in the charter.

Reply:
Nothing to add to what @iherman wrote.

@OR13 any objection to resolving this conversation?

Reply:
I'd prefer not to give the working group a blank check on notes, based on the previous experience we have had, the notes become a dumping ground for lack of working group consensus. 

That in turn harms the quality of the other deliverables by comparison to the notes. 

For example, a did method could be published as a note, and then later made a recommendation.

I think it's better to have a charter that is narrow, specific, and where each item has the full support of the working group.

Allowing fractions of the group to publish notes that don't reflect working group consensus seems like something that will harm our primary objective, and we have an active CG that can publish community drafts, which don't require full working group consensus.

Reply:
@rxgrant, I interpret what I see in the comments as a fundamental mistrust in the management of the group with regard to the consensus seeking process, at least for the acceptance of notes being published. I sincerely hope I completely misunderstand this.

However. If the intention is to strongly limit the ability of this group to publish unforeseen notes, then I believe there are two things that must be done:

1. List in the deliverables _what_ notes are allowed to be published
2. Add to the 'out of scope' section that the group is _not_ allowed to publish anything else, including wg notes.

The reason the second point is relevant is that such a restriction is highly unusual in working groups and, therefore, should be made explicit.

But it may well be (and I hope!) that I misunderstand the discussion here.

Reply:
One more remark:

@OR13 said:

> For example, a did method could be published as a note, and then later made a recommendation.

The process makes this more difficult, actually. The latest version says:

> A Group Note (NOTE) is published to provide a stable reference for a useful document that is not intended to be a formal standard. [See [2023 process document](https://www.w3.org/2023/Process-20230612/#Note)]

Turning a note into a Recommendation would require a separate chartering step for a new WG and the AC might as well refuse to accept this if it simply comes directly from a Note.

Reply:
> @rxgrant, I interpret what I see in the comments as a fundamental mistrust in the management of the group with regard to the consensus seeking process, at least for the acceptance of notes being published. I sincerely hope I completely misunderstand this.

@iherman  It is absolutely approriate to define where consensus will be reached in the chartering process.  If you think chartering is not part of the consensus process, and that it only begins later, then I hope that you will clarify further.

Reply:
@rxgrant I believe you misunderstood. I was referring to the fact that whether a WG decides to publish (or not) a particular WG Note is subject to group consensus. Therefore, I am not sure that it is worth saying more than the proposed generic text in the charter, namely that the WG may publish WG notes.

If the group has a consensus in making the charter more strict, then be it.

Reply:
I don't really have a strong opinion whether notes should be in scope or not for this WG, I'm fine either way. But referring to them as a "dumping group" or "disastrous process" isn't helpful. Which note(s) from a past WG are you referring to, and what was wrong in those cases?

Reply:
The DID Spec Registries is a WG note.

I don't believe it has gone well.

Almost every new method registered has JSON-LD / RDF issues with it.

But the registration process does not require well formed documents.

Perhaps the new process has addressed the previous challenges the DID WG faced, which led us to publish the JSON-LD context, JSON-LD extension points, and the DID Method Registry as a WG note.

I'm sorry my maintainer experience here requires me to describe the note as a "dumping ground", but thats what it is, since the maintainers of the registry have little ability to ensure quality, and the working group is not "coming to consensus" to merge the PRs to this note... Its rare that we get more than 1 or 2 reviews of a new PR here.

Reply:
> The DID Spec Registries is a WG note.

I think the method registry part has indeed been problematic at times and definitely a lot of work for maintainers. But other parts like extension parameters and extension metadata properties have been really useful in my opinion...

Reply:
@iherman 

> > Allowing fractions of the group to publish notes that don't reflect working group consensus seems like something that will harm our primary objective, and we have an active CG that can publish community drafts, which don't require full working group consensus.
> 
> I do not understand this argument. The very act of publishing a note _**does**_ require group consensus, i.e., a formal resolution of the Working Group.

Do you agree or disagree with [this statement](https://github.com/w3c/did-imp-guide/pull/27/#issuecomment-918745249) by the Chair, which states that the contents of Notes is not subject to group consensus?

Reply:
@rxgrant,

I believe you compare two different things. And I realize that this is all a bit legalistic.

- To be precise, what I said is that it requires a Working Group consensus to register a completely new entry in the formal collection of W3C's publications, a.k.a. a document with a `https://www.w3.org/TR/XXX` URL, where `XXX` is what the jargon calls the "short name" of the publication. A good example is the [CBOR Note](https://www.w3.org/TR/did-cbor-representation/) whose short name is `did-cbor-representation`. This required a [formal resolution](https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-06-15-did#resolution1) of the WG on its [meeting on the 15th of June '21](https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-06-15-did#section2).

    Note that the resolution also included the approval for the usage of "echidna" for automatic publishing, ie, the publication of a new version of that Note as a result of a github PR. That is possible because, though each version of that Note is published as a "new" Note (as @brentzundel said in https://github.com/w3c/charter-drafts/pull/448#discussion_r1378653120), it is not a new entry in the W3C publication list, hence this automatic update is possible as far as W3C is concerned. (The group could have decided to require a formal WG resolution for the publication of each new version. It didn't.)
- _The content_ of the note does not require 100% consensus for all parties involved in the WG; this depends on the nature of the Note itself. The requirements are fairly loose, although it is indeed good practice to seek for consensus. (After all, per [process](https://www.w3.org/2023/Process-20230612/#note-track), "A Group Note (NOTE) is published to provide a stable reference for a useful document ***that is not intended to be a formal standard.***" Emphasis is mine.)

Reply:
The above https://github.com/w3c/charter-drafts/pull/448#discussion_r1381888017 is edited to remove an overstatment about whether the policy was applied evenly.

Reply:
Resolving this conversation, as the discussion about notes has drifted into other conversations in this PR.